mirage/seed_image/catalog_seed_image.py

```diff
@@ -2033,6 +2033,15 @@ def makePos(self, alpha1, delta1):
         delta2 = "%1s%2.2d:%2.2d:%7.4f" % (sign, decd, decm, decs)
         alpha2 = alpha2.replace(" ", "0")
         delta2 = delta2.replace(" ", "0")
+
+        # The following fixes a format issue; it is not clear why the signa
```

Comment:
Did you see a case where this was needed? This was in `catalog_seed_generator` a long time ago, but was removed when it was determined it wasn't needed.

Reply:
Following @KevinVolkSTScI comment I will remove these lines.